DepotDownloader/Util.cs

```diff
@@ -119,6 +120,31 @@ public static byte[] AdlerHash(byte[] input)
             return BitConverter.GetBytes(a | (b << 16));
         }
 
+        public static byte[] FileSHAHash(string filename)
+        {
+            using (var fs = File.Open(filename, FileMode.Open))
+            using (var sha = SHA1.Create())
+            {
+                var output = sha.ComputeHash(fs);
+
+                return output;
+            }
+        }
+
+        public static bool SaveManifestToFile(DepotManifest manifest, string filename)
```

Comment:
FYI I added SaveToFile back in SK in last beta.

Reply:
Ok, should I want until SteamKit is bumped?

Reply:
Kinda of can't update SK standalone without the refactor done in this PR because I changed checksums from byte[] to int lol.

So bump SK in this pr.